plugins/angular.js

```diff
@@ -65,3 +86,5 @@ angularPlugin._normalizeData = function (data) {
 };
 
 module.exports = angularPlugin;
+// Install plugin when included
+angularPlugin(window.Raven);
```

Comment:
Really good point with catching uncaughtExceptions. 

What I have done in one project was to handle `config`, `install` and `addPlugin` in my build script with gulp. But that adds a lot of moving external parts to maintain for you.
If you want, I could share my build as an example.

In that case I think we should remove `config` and `install` from RavenProvider and revert [line 90](821/files#diff-8781144d718c7370adbb09707bf3e448R90)